### PR TITLE
Remove dependency on usethis

### PR DESCRIPTION
**Pull request by olivroy**
Also

* Update gh actions with `usethis::use_tidy_github_actions()`.
* Remove deprecation warnings from ggplot2 3.4.0, tibble 2.0.0 (likely fixing #11, if I understood correctly that the matrix always has 2 columns), and tidyselect 1.2.0.
* use `rlang::check_installed()` to check for missing Suggests. It works really well as it offers a prompt for the user to install missing suggests in interactive sessions (rather than errror)
* Use cli functions instead of usethis. https://cli.r-lib.org/articles/usethis-ui.html

Let me know if you have questions

Cheers!

### CODE REVIEW

File: R/outliers.R [hunk 1]
```diff
@@ -257,13 +258,13 @@ detect_outliers.incremental <- function(
       dplyr::filter(outlier == "yes")
 
     if(nrow(verbose_vector) == 0) {
-      usethis::ui_done(x = "No outliers found.")
+      cli::cli_alert_success(x = "No outliers found.")
```
Comment (olivroy) [inline R/outliers.R:261]:
```suggestion
      cli::cli_alert_success("No outliers found.")
```

arg is text

**Review by olivroy — COMMENTED** [review]
remove x

**Comment by fmmattioni** [comment]
Dear @olivroy, thanks for the PR! Looks great!

**Comment by olivroy** [comment]
Glad you like it! Let me know if something is not working as expected or if you have questions!

**Pull request merged**